src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -60,6 +60,7 @@ function SearchPageBottomTab() {
                             activeWorkspaceID={policyID}
                             breadcrumbLabel={translate('common.search')}
                             shouldDisplaySearch={false}
+                            isCustomSearchQuery={queryJSON.inputQuery !== SearchUtils.buildCannedSearchQuery()}
```

Comment:
This will display the `Cancel` button for any of the tabs or any of the non-expense categories (Chats, Invoices, Trips). Was that the intention, or maybe it makes more sense to only show it for filters that aren't any of those categories?

Reply:
Yes good point, I've fixed it [here](https://github.com/Expensify/App/pull/48566/commits/aacab14025590c8ca97f707f59250c24c00ab9f2)